src/renderers/webgl/WebGLTextures.js

```diff
@@ -10,8 +10,7 @@ function WebGLTextures( _gl, extensions, state, properties, capabilities, utils,
 	const maxCubemapSize = capabilities.maxCubemapSize;
 	const maxTextureSize = capabilities.maxTextureSize;
 	const maxSamples = capabilities.maxSamples;
-	const hasMultisampledRenderToTexture = extensions.has( 'WEBGL_multisampled_render_to_texture' );
-	const MultisampledRenderToTextureExtension = hasMultisampledRenderToTexture ? extensions.get( 'WEBGL_multisampled_render_to_texture' ) : undefined;
+	const MultisampledRenderToTextureExtension = extensions.has( 'WEBGL_multisampled_render_to_texture' ) ? extensions.get( 'WEBGL_multisampled_render_to_texture' ) : null;
```

Comment:
I think this is equivalent (`gl.getExtension()` returns `null` when not available):

```js
const MultisampledRenderToTextureExtension = extensions.get( 'WEBGL_multisampled_render_to_texture' );
```

Reply:
Yes, `get()` will return `null` if the extension is not available. However, there is actually a small difference. `get()` will log a warning when an extension is not available which can be quite confusing for users. Hence, it's better to always check if an extension is available via `has()` and only then request it via `get()`.